### PR TITLE
Feature/icky 307 zip code : Store finder by zip code implemented

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

File: components/LoginModal.vue [hunk 1]
```diff
@@ -162,7 +163,13 @@ export default {
     SfLoader,
   },
   setup() {
+    interface loginForm {
```
Comment (kibo-kevinwatts) [inline components/LoginModal.vue]: dont define interfaces/types in components

File: composables/storeFinder/usePurchaseLocation.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const usePurchaseLocation = () => {
         })
         purchaseLocation.value = response.data.spLocations.items[0]
       } catch (error) {
-        console.log(error)
+        purchaseLocation.value = {}
```
Comment (kibo-kevinwatts) [inline composables/storeFinder/usePurchaseLocation.ts]: Shouldn't this be         purchaseLocation.value = null on error?

Comment (kibo-chandradeeptalaha) [reply composables/storeFinder/usePurchaseLocation.ts]: Done

File: components/LoginModal.vue (hunk 1 above)
Comment (kibo-chandradeeptalaha) [reply components/LoginModal.vue]: defined inside type folder in components.

**Pull request closed without merging**